dashboard/client/src/pages/actor/index.tsx

```diff
@@ -14,52 +10,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Represent the standalone actors page.
+ */
 const Actors = () => {
   const classes = useStyles();
-  const [autoRefresh, setAutoRefresh] = useState(true);
-  const [actors, setActors] = useState<{ [actorId: string]: Actor }>({});
-  const [timeStamp, setTimeStamp] = useState(dayjs());
-  const queryActor = () =>
-    getActors().then((res) => {
-      if (res?.data?.data?.actors) {
-        setActors(res.data.data.actors);
-      }
-    });
-
-  useEffect(() => {
-    let tmo: NodeJS.Timeout;
-    const refreshActor = () => {
-      const nowTime = dayjs();
-      queryActor().then(() => {
-        setTimeStamp(nowTime);
-        if (autoRefresh) {
-          tmo = setTimeout(refreshActor, API_REFRESH_INTERVAL_MS);
-        }
-      });
-    };
-
-    refreshActor();
-
-    return () => {
-      clearTimeout(tmo);
-    };
-  }, [autoRefresh]);
 
   return (
     <div className={classes.root}>
-      <TitleCard title="ACTORS">
-        <Grid container alignItems="center">
-          <Grid item>
-            Auto Refresh:{" "}
-            <Switch
-              checked={autoRefresh}
-              onChange={({ target: { checked } }) => setAutoRefresh(checked)}
-            />
-          </Grid>
-          <Grid item>{timeStamp.format("YYYY-MM-DD HH:mm:ss")}</Grid>
-        </Grid>
-        <ActorTable actors={actors} />
-      </TitleCard>
+      <TitleCard title="ACTORS">{<ActorList />}</TitleCard>
```

Comment:
```suggestion
      <TitleCard title="ACTORS"><ActorList /></TitleCard>
```

Reply:
further suggestion is to break and indent child component on new line if the linter doesn't enforce it already